FAIL ./yatze.test.ts
  Chance
    ✓ roll 5 dice (2 ms)
  Yatze
    ✓ roll 5 dice successfully (1 ms)
    ✓ roll 5 dice unsuccessfully
  Ones
    ✕ roll 5 dice successfully (2 ms)

  ● Ones › roll 5 dice successfully

    expect(received).toEqual(expected) // deep equality

    Expected: 2
    Received: 12

      20 | describe('Ones', () => {
      21 |   it('roll 5 dice successfully', () => {
    > 22 |     expect(roll([1,1,2,4,4],"Ones")).toEqual(2);
         |                                      ^
      23 |   })
      24 | });

      at Object.<anonymous> (yatze.test.ts:22:38)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 3 passed, 4 total
Snapshots:   0 total
Time:        2.712 s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ test: `jest --maxWorkers=2`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sandbox/.npm/_logs/2025-05-04T21_20_57_752Z-debug.log
